packages/lesswrong/lib/collections/tags/helpers.ts

```diff
@@ -141,3 +142,13 @@ export const tagRouteWillDefinitelyReturn200 = async (req: Request, res: Respons
 export const EA_FORUM_COMMUNITY_TOPIC_ID = 'ZCihBFp5P64JCvQY6';
 export const EA_FORUM_TRANSLATION_TOPIC_ID = 'f4d3KbWLszzsKqxej';
 export const EA_FORUM_APRIL_FOOLS_DAY_TOPIC_ID = '4saLTjJHsbduczFti';
+
+export const isTagAllowedType3Audio = (tag: TagPageFragment|DbTag): boolean => {
+  if (!allowTypeIIIPlayerSetting.get()) return false
+
+  return !!tag.forceAllowType3Audio && !!tag.description && !tag.deleted
```

Comment:
Decided to go with manual flag rather than minimum description length?  (Seems fine to me.)

Reply:
Yeah, Oli said all we're doing for now is enabling for just those pages